CONTRIBUTING.md

```diff
@@ -1,52 +1,74 @@
 # Contributing to OpenEBS
 
-Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [Overview](./contribute/design/README.md)
+Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [overview](./contribute/design/README.md)
 
-OpenEBS is innovation in OpenSource. You are welcome to contribute in any way you can and all the help you can provide is very much appreciated. 
+Firstly, if you are unsure or afraid of anything, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions and don't want a wall of rules to get in the way of that.
 
-- Raise Issues on either the functionality or documentation
-- Submit Changes to Improve Documentation 
-- Submit Proposals for new Features/Enhancements 
-- Submit Changes to the Source Code
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover all the points we're looking for in your contributions, raising your chances of quickly merging or addressing your contributions.
+
+That said, OpenEBS is an innovation in open-source. You are welcome to contribute in any way you can and all the help provided is very much appreciated. 
+
+- Raise issues to request new functionality, fix documentation or for reporting bugs.
+- Submit changes to improve documentation. 
+- Submit proposals for new features/enhancements.
+- Submit fixes for bugs reported/found. 
+- Submit changes to the source code.
+- Solve existing issues.
```

Comment:
I think the last three statements mean the same. How about combining them as:
- Solve existing issues related to documentation or code.

Reply:
And, each of those above lines can be linked to the corresponding subsections that follow.

Reply:
Implemented.